content/en/core/overview/self-hosting-single-node.md

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
Please move this to beside the other hosting docs here: https://docs.communityhealthtoolkit.org/apps/guides/hosting/4.x/